var/spack/repos/builtin/packages/py-pylint/package.py

```diff
@@ -43,33 +44,37 @@ class PyPylint(PythonPackage):
     depends_on("python@3.6:", when="@2.8.2:", type=("build", "run"))
     depends_on("python@3.6.2:", when="@2.13.5:", type=("build", "run"))
     depends_on("python@3.7.2:", when="@2.14.0:", type=("build", "run"))
+    # depends_on('py-setuptools-scm@1.15.0:', type='build')
```

Comment:
Do you know why this comment was originally added?

Reply:
No idea.

Reply:
Looks like it's been around for 5+ years, you can just remove it